src/components/forms/book-club-manage-member.form.tsx

```diff
@@ -271,7 +318,30 @@ const BookClubManageMemberForm = ({
           ...(!!coalescedMembership.departed && styles.removedMemberCell)
         }}
       >
-        DOIT
+        {coalescedMembership.isOwner ? (
```

Comment:
This file is too long; Pull the make owner button (including tooltip and instantiated dialog into a new component.